### ISSUE TITLE
Set up Netlify integration for review apps

### ISSUE THREAD

**Issue by tnir** [kubernetes/kubernetes-docs-ja#20]
**This is a...** 
- [x] Feature Request
- [ ] Bug Report

**Problem:**

Hard to complete to review in markdown form in GitHub PR.

**Proposed Solution:**

Use review apps with Netlify integration.

**Comment by tnir** [comment]
I and @lucperkins found the following error at master:

```
ERROR 2018/08/31 16:37:04 error processing shortcode "shortcodes/code.html" for page "docs/tasks/administer-cluster/quota-api-object.md": template: shortcodes/code.html:14:16: executing "shortcodes/code.html" at <readFile $filename>: error calling readFile: stat /opt/build/repo/content/cn/docs/tasks/administer-cluster/quota-objects-pvc.yaml: no such file or directory
ERROR 2018/08/31 16:37:04 error processing shortcode "shortcodes/code.html" for page "docs/tasks/administer-cluster/quota-api-object.md": template: shortcodes/code.html:14:16: executing "shortcodes/code.html" at <readFile $filename>: error calling readFile: stat /opt/build/repo/content/cn/docs/tasks/administer-cluster/quota-objects-pvc-2.yaml: no such file or directory

Total in 3073 ms
```

@nasa9084 Do you know about this? We confirmed k/website `release-1.10` works 👍

**Comment by nasa9084** [comment]
I think because content/cn directory is not in this repository.

**Comment by tnir** [comment]
Resolved by #27 after some investigations

/close
